src/controller/CommissioningDelegate.h

```diff
@@ -417,6 +437,7 @@ class CommissioningParameters
     Optional<bool> mAttemptWiFiNetworkScan;
     Optional<bool> mAttemptThreadNetworkScan; // This automatically gets set to false when a ThreadOperationalDataset is set
     Optional<bool> mSkipCommissioningComplete;
+    Optional<bool> mCheckForMatchingFabric;
```

Comment:
Might be worth documenting why this is an `Optional<bool>` instead of just a bool.  For some of the other booleans, there is default auto-guessing behavior based on other information, but for this one the default is just false, right?

Reply:
checked this to be just a bool